src/Illuminate/Auth/Middleware/RequirePassword.php

```diff
@@ -50,11 +50,12 @@ public function __construct(ResponseFactory $responseFactory, UrlGenerator $urlG
      * @param  \Illuminate\Http\Request  $request
      * @param  \Closure  $next
      * @param  string|null  $redirectToRoute
+     * @param  int|null  $passwordTimeoutSeconds
      * @return mixed
      */
-    public function handle($request, Closure $next, $redirectToRoute = null)
+    public function handle($request, Closure $next, $redirectToRoute = null, int $passwordTimeoutSeconds = null)
```

Comment:
This is a breaking change to the method signature.

Reply:
Thanks for fast response. One small question, why though? I'm adding optional argument to method.

Reply:
If a class is extending this method it'll break.